### DISCUSSION TITLE
Janet 1.34.0

### DISCUSSION THREAD

**Discussion by bakpakin** [janet-lang/janet#1436] [General]
## What's Changed
* add a new (split) PEG special by @ianthehenry in https://github.com/janet-lang/janet/pull/1346
* Add buffer/push-* sized int and float by @pnelson in https://github.com/janet-lang/janet/pull/1359
* Update CHANGELOG.md by @pepe in https://github.com/janet-lang/janet/pull/1360
* Cosmetically tweak module/expand-path docstring by @sogaiu in https://github.com/janet-lang/janet/pull/1364
* Address #1370 by @sogaiu in https://github.com/janet-lang/janet/pull/1373
* Fix typo in destructuring by @pepe in https://github.com/janet-lang/janet/pull/1371
* Tweak debug/stacktrace docstring (#1365) by @sogaiu in https://github.com/janet-lang/janet/pull/1367
* Tweak fiber/last-value docstring by @sogaiu in https://github.com/janet-lang/janet/pull/1389
* Add to propagate docstring (#1365) by @sogaiu in https://github.com/janet-lang/janet/pull/1392
* Fix: make `proc_get_status` compliant to OS documentation. by @llmII in https://github.com/janet-lang/janet/pull/1403
* Swap set / body order for each (#1400) by @sogaiu in https://github.com/janet-lang/janet/pull/1402
* Improve documentation on subprocess API by @amano-kenji in https://github.com/janet-lang/janet/pull/1394
* Fix documentation of peg/replace by @MaxGyver83 in https://github.com/janet-lang/janet/pull/1414
* Doc tweaks for ev/deadline and ev/with-deadline by @sogaiu in https://github.com/janet-lang/janet/pull/1410
* Prepare for 1.34.0 by @pepe in https://github.com/janet-lang/janet/pull/1429
* Add exists test for dist directory on build command clean by @pepe in https://github.com/janet-lang/janet/pull/1430
* Add additional format options for os/clock by @wishdev in https://github.com/janet-lang/janet/pull/1432

## New Contributors
* @pnelson made their first contribution in https://github.com/janet-lang/janet/pull/1359
* @amano-kenji made their first contribution in https://github.com/janet-lang/janet/pull/1394
* @MaxGyver83 made their first contribution in https://github.com/janet-lang/janet/pull/1414
* @wishdev made their first contribution in https://github.com/janet-lang/janet/pull/1432

**Full Changelog**: https://github.com/janet-lang/janet/compare/v1.33.0...v1.34.0

<hr /><em>This discussion was created from the release <a href='https://github.com/janet-lang/janet/releases/tag/v1.34.0'>Janet 1.34.0</a>.</em>

**Comment by afranchuk** [comment]
Thanks for all the hard work! Two questions:
* Why no pre-built binaries for linux/windows?
* When will the documentation on janet-lang.org be updated (it seems to be 1.32)?

**Comment by pyrmont** [comment]
@bakpakin The absence of pre-built binaries for Linux is causing GitHub workflows that depend on [action-janet-test](https://github.com/pyrmont/action-janet-test) to break. Could these be added back?

**Reply by bakpakin** [reply]
Yes, something messed up here, looking into this.

**Reply by pyrmont** [reply]
Thanks for adding these back!
